apps/test/unit/gamelab/spritelab/spriteUtilsTest.js

```diff
@@ -0,0 +1,100 @@
+/* global p5 */
+import {expect} from '../../../util/configuredChai';
+import createGameLabP5 from '../../../util/gamelab/TestableGameLabP5';
+import {sandboxDocumentBody} from '../../../util/testUtils';
+import * as spriteUtils from '@cdo/apps/gamelab/spritelab/spriteUtils';
+
+describe('Sprite Utils', () => {
+  let gameLabP5, createSprite, animation;
+  // Using the aggressive sandbox here because the P5 library generates
```

Comment:
aggressive sandbox?

Reply:
I'm not entirely sure, I was using GameLabSpriteTest as a guide. But it doesn't seem to be necessary so I deleted it.